experimental/dds/tree2/src/feature-libraries/schemaIndexFormat.ts

```diff
@@ -29,31 +29,13 @@ import { brand, fail } from "../util";
 
 const version = "1.0.0" as const;
 
-/**
- * Create a TypeBox string schema for a branded string type.
- * This only validates that the value is a string,
- * and not that it came from the correct branded type (that information is lost when serialized).
- */
-function brandedString<T extends string>(): TUnsafe<T> {
-	// This could use:
-	// return TypeSystem.CreateType<T>(name, (options, value) => typeof value === "string")();
-	// Since there isn't any useful custom validation to do and
-	// TUnsafe is documented as unsupported in `typebox/compiler`,
-	// opt for the compile time behavior like the above, but the runtime behavior of the built in string type.
-	return Type.String() as unknown as TUnsafe<T>;
-}
-
-const FieldKindIdentifier = brandedString<FieldKindIdentifier>();
-const TreeSchemaIdentifier = brandedString<TreeSchemaIdentifier>();
-const LocalFieldKey = brandedString<LocalFieldKey>();
-const GlobalFieldKey = brandedString<GlobalFieldKey>();
-
 const FieldSchemaFormat = Type.Object(
 	{
 		kind: FieldKindIdentifier,
 		types: Type.Optional(Type.Array(TreeSchemaIdentifier)),
 	},
-	{ additionalProperties: false },
+	// TODO: Investigate behavioral change here due to typebox update.
```

Comment:
this is due to discourse in https://github.com/sinclairzx81/typebox/issues/373. Now that I've tracked that down, it's probably worth going and adding { additionalProperties: false } to edit schema where appropriate and revisiting what's actually going on here in a separate PR.